README.md

```diff
@@ -1,31 +1,52 @@
 # MONTAGE
 
-To install:
-1. First create an rsa key pair:
+## Initial Setup
+
+1. git clone --recursive https://github.com/NGS2Montage/montage.git
+
+2. Navigate into the project folder, then create an rsa key pair:
     ```
     openssl genpkey -algorithm RSA -pkeyopt rsa_keygen_bits:4096 -out private.pem
     openssl rsa -in private.pem -outform PEM -pubout -out public.pem
     ```
-2. Start a virtual environment
-3. Then run: 
+3. Start a virtual environment
```

Comment:
There are several ways to accomplish this. My preferred way is 
```
mkvirtualenv montage_venv --python=`which python3`
```

Reply:
ok this is good learning for me
I will work on this later and then update the readme file accordingly
https://virtualenvwrapper.readthedocs.io/en/latest/install.html
Thank you